apps/src/templates/studentSnapshot/StudentSnapshot.tsx

```diff
@@ -38,6 +59,8 @@ const StudentSnapshot: React.FC = () => {
   const [isLessonsLoading, setIsLessonsLoading] = useState<boolean>(false);
   const [hasUnnumberedLessons, setHasUnnumberedLessons] =
     useState<boolean>(false);
+  const [cfuLevels, setCfuLevels] = useState<CFULevel[]>([]);
```

Comment:
Do you think we will want these CFU levels in multiple widgets or should we put this logic into the CFU widget?

Fine if we want to keep it this way for now and move it later.